src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -150,6 +153,33 @@ static int client_receive_message_udp(
                 void *userdata);
 static void client_stop(sd_dhcp_client *client, int error);
 
+int sd_dhcp_client_id_to_string(char **ret, const void *data, size_t len) {
```

Comment:
`sd_` means that this is a public function, so parameters needs to be verified.
At least do
```
assert_return(ret, -EINVAL);
assert_return(data, -EINVAL);
```
Also `len` is not used. Shouldn't there be a check that `len` is big enough for the operations below?

Reply:
done